tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,13 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+#@pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    
+    assert result is False
```

Comment:
Excellent assert! It would also be a good idea to check to see if the inventory list is still the correct length of 3 and nothing got removed by accident.